### PR TITLE
[REF] runbot_travis2docker: Skip builds if commit message has [ci skip] (#117)

### PR DESCRIPTION
**Pull request by hugho-ad**
This is  a cherry-pick of this commit
https://github.com/OCA/runbot-addons/commit/34a7beec1cfabe75423f5c27607ac51c332efafa

### CODE REVIEW

**Comment by hugho-ad** [comment]
@moylop260 here is the cherry-pick

Regards

**Comment by moylop260** [comment]
Could you force other commit to build travis?

**Comment by moylop260** [comment]
Could you check the red status?

**Comment by zaoral** [comment]
+1 for bring this functionality to vauxoo :blush:

**Comment by osvalr** [comment]
this is kinda related to this one reported a while ago in another repo: https://github.com/Vauxoo/maintainer-quality-tools/issues/96
:+1: for me

**Comment by JesusZapata** [comment]
@moylop260 
This PR skip the runbot build if in the commit message have `[ci skip]`

File: runbot_travis2docker/models/runbot_build.py [hunk 1]
```diff
@@ -135,10 +137,9 @@ def job_10_test_base(self, cr, uid, build, lock_path, log_path):
         if not build.branch_id.repo_id.is_travis2docker_build:
             return super(RunbotBuild, self).job_10_test_base(
                 cr, uid, build, lock_path, log_path)
-        if not build.docker_image or not build.dockerfile_path \
-                or build.result == 'skipped':
-            _logger.info('docker build skipping job_10_test_base')
-            return MAGIC_PID_RUN_NEXT_JOB
+        skip = self.skip_check(build)
+        if skip:
+            return skip
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_build.py:142]: We have a skip status, we could use it in order to skip just the firts job method

**Pull request closed without merging**